### PR TITLE
Add new adapter for RTB House

### PR DESCRIPTION
**Pull request by CapedHero**
Hello there,

Please merge a new bidder adapter for RTB House.

This follows guidelines from http://prebid.org/prebid-server/developers/add-new-bidder.html

Thank you for your time and comments!

### CODE REVIEW

File: adapters/rtbhouse/rtbhouse.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+		}
+		return nil, []error{err}
+	}
+
+	var openRTBBidderResponse openrtb.BidResponse
+	if err := json.Unmarshal(bidderRawResponse.Body, &openRTBBidderResponse); err != nil {
+		return nil, []error{err}
+	}
+
+	bidderResponse = adapters.NewBidderResponseWithBidsCapacity(1)
```
[the remark this answers is not in the document]
Comment (CapedHero) [reply adapters/rtbhouse/rtbhouse.go]: Thank you for your comment. I will go with the dynamic version based on `len`!

File: adapters/rtbhouse/rtbhouse.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+type RTBHouseAdapter struct {
+	endpoint string
+}
+
+// MakeRequests prepares the HTTP requests which should be made to fetch bids.
+func (adapter *RTBHouseAdapter) MakeRequests(
+	openRTBRequest *openrtb.BidRequest,
+) (
+	requestsToBidder []*adapters.RequestData,
+	errors []error,
```
[the remark this answers is not in the document]
Comment (CapedHero) [reply adapters/rtbhouse/rtbhouse.go]: I agree with your remark. I will push an update.

**Comment by mansinahar** [comment]
Thanks for the PR @CapedHero !

**Comment by mansinahar** [comment]
@CapedHero can you please resolve the conflicts on this one when you have a chance? We can go ahead and merge it then. Thanks!

**Comment by CapedHero** [comment]
@mansinahar Consider it done :)

**Comment by CapedHero** [comment]
Is it possible to merge PR today? Beginning Monday next week, I will be away from the office for a couple of days, so it will be hard for me to reply to any queries, resolve conflicts, etc.

**Comment by mansinahar** [comment]
@CapedHero Merged!

**Pull request merged**